README.md

```diff
@@ -16,9 +16,9 @@ addressed.
 
 FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) and [LibRedirect](https://github.com/libredirect/libredirect) extension, which will allow you to open YouTube links into FreeTube. You must enable the option within the advanced settings for it to work.
```

Comment:
extensions instead of extension

This was missing on the previous pr. Could u address it with a commit.

Reply:
May I do that in another pull request? Kind of different type of change. ;)

Reply:
Different pull requests and commit messages are only really necessary if you can actually tell the commits apart. As your commit messages are just "Update README.md", you might as well just stick the changes in this one.

Reply:
Oops, I send this patch directly on GitHub website and didn't notice that my words are in the body, should be more precisely in the subject, just updated it!

Reply:
please address this there is no need for a separate pr